data/xfstests/wrapper.sh

```diff
@@ -90,7 +94,7 @@ fi
 
 # Run test
 log_file="/tmp/xfstests-$(echo "$1" | tr '/' '_').tmp"
-./$(basename "$PROG") -d "$1" | tee "$log_file"
+./$(basename "$PROG") -d "$OPTIONS" "$1" | tee "$log_file"
```

Comment:
This will add one more space if $OPTIONS="". Although I know this doesn't matter

Reply:
Yes, I aware of this, a duplicate space will not harm to the original command.